Extensions/CLIExtensionTemplate.json

```
@@ -0,0 +1,8 @@
+{
+    "build_subdirectory": ".",
+    "category": "Examples",
+    "depends": [],
```

consider being more explicit:

```suggestion
    "build_dependencies": [],
```